renovate.json

```
@@ -117,6 +117,13 @@
       ],
       "registryUrlTemplate": "https://{{{registryUrl}}}",
       "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": ["^\\.github/workflows/commitlint\\.yaml$"],
+      "matchStrings": [
+        "npm install --save-dev @(?<depName>.*?)@(?<currentValue>.+)"
```

```suggestion
        "npm install --save-dev (?<depName>@?.*?)@(?<currentValue>.+)"
```
The @ needs to be inside depname:

https://github.com/Racer159/uds-core/issues/2
https://github.com/Racer159/uds-core/blob/main/renovate.json